backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -2,6 +2,49 @@ $padding-x-navbar: 26px;
 $padding-y-navbar: 13px;
 $padding-y-navbar-submenu: 9px;
 
+.menu-button-icon {
+  width: 34px;
+  height: 32px;
+  margin-right: 10px;
+  background: $color-btn-bg;
+  color: $color-btn-text;
+  border-radius: 4px;
+  text-align: center;
+  cursor: pointer;
+  display: none;
+  &:hover {
+    background: $color-btn-hover-bg;
+  }
+  @include media($tablet) {
+    display: block;
+  }
+
+  i {
+    padding-top: 7px;
+    color: $color-btn-text;
+    font-size: 18px;
+  }
+}
+
+#menu-button {
+  display: none;
+
+  &:checked ~ .admin-wrapper {
+    overflow: hidden;
+
+    .admin-nav {
+      transform: translateX(0);
+    }
+
+    .admin-content  {
+      transform: translateX($width-sidebar);
+      @include media($large-device) {
+        transform: none;
+      }
+    }
+  }
+}
+
```

Comment:
Actually we tried to add a CSS transition to the nav, but while using only CSS allows you to have a cleaner code, you don't have the same power as you have with Javascript :) 

We found a bug in particular that occurs while resizing the window, as explained by this GIF:

![transition-menu-bug](https://cloud.githubusercontent.com/assets/1730394/23160393/9555a766-f826-11e6-8cb5-dd1073cc429b.gif)

So we thought that in order to avoid focusing too much on the transition effect, we'd implement a first version of the collapsible navigation without effect.

Any suggestion to avoid this bug?